timeseer/podman/roles/timeseer/handlers/main.yml

```
@@ -0,0 +1,17 @@
+- name: Signal daemon Restart
+  ansible.builtin.set_fact:
+    daemon_reload: true
+  when: item.changed
+  loop: "{{ timeseer_j2.results }}"
+  loop_control:
+    label: "{{ item.dest }}"
+
+- name: Reload systemd daemon
+  ansible.builtin.systemd:
+    daemon_reload: "{{ daemon_reload }}"
+  when: daemon_reload | default(false) | bool
+
+- name: Restart timeseer service
+  ansible.builtin.systemd:
+    name: ts-container
```

This name does not correspond to the name of the container.


```suggestion
    name: timeseer
```